pkg/performanceprofile/profilecreator/helper.go

```diff
@@ -4,15 +4,9 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+//nolint:unused
```

Comment:
then let's remove it

Reply:
I can't. 
linter mark it as "not used" because it is only used by source files in the "test" folder which I have marked for the linter to ignore. 

adding "test" folder is somehow problematic, not sure why (yet) but linter complains about some variables to be undefined
```
test/e2e/basic/available.go:1: : # github.com/openshift/cluster-node-tuning-operator/test/e2e/basic
test/e2e/basic/available.go:31:53: undefined: cs
test/e2e/basic/available.go:35:52: undefined: cs
test/e2e/basic/available.go:42:14: undefined: cs
test/e2e/basic/custom_node_labels.go:45:38: undefined: cs
test/e2e/basic/custom_node_labels.go:51:37: undefined: cs
test/e2e/basic/custom_node_labels.go:56:45: undefined: cs
test/e2e/basic/custom_pod_labels.go:45:38: undefined: cs
test/e2e/basic/custom_pod_labels.go:51:36: undefined: cs
test/e2e/basic/custom_pod_labels.go:56:45: undefined: cs
test/e2e/basic/default_irq_smp_affinity.go:53:38: undefined: cs
test/e2e/basic/default_irq_smp_affinity.go:53:38: too many errors (typecheck)
package e2e

```

Reply:
ok, let's keep it like this and improve later, this PR is already big and painful as it is now.